### PR TITLE
Add frontmatter to contributor docs

### PR DESCRIPTION
**Pull request by asmacdo**
I found a few files in the contributor docs that were not included in the website.

### CODE REVIEW

File: website/content/en/docs/contribution-guidelines/developer-guide.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# Operator SDK Developer guide
+---
+title: Development Environment
+weight: 30
+---
 
 This document explains how to setup your dev environment.
```
Comment (camilamacedo86) [inline website/content/en/docs/contribution-guidelines/developer-guide.md:7]: It is missing the info over how to test Ansible and Helm locally. See: https://github.com/operator-framework/operator-sdk/blob/master/doc/dev/developer_guide.md#how-to-test-the-changes-done-for-ansiblehelm-based-operator-projects

File: website/content/en/docs/contribution-guidelines/local-docs.md [hunk 2]
```diff
@@ -24,7 +24,8 @@ $ git submodule update --init --recursive
 
 ## Build and Serve
 
-You can build and serve your docs to localhost:1313 with:
+You can build and serve your docs to localhost:1313. From the `website`
+directory run:
```
Comment (camilamacedo86) [inline website/content/en/docs/contribution-guidelines/local-docs.md:29]: The links in https://deploy-preview-2783--operator-sdk.netlify.com/docs/contribution-guidelines/developer-guide/ are not working.

Comment (asmacdo) [reply website/content/en/docs/contribution-guidelines/local-docs.md:29]: Should be fixed. There are still a lot of broken links all over which I plan to fix along with the new link checker.

File: website/content/en/docs/contribution-guidelines/developer-guide.md (hunk 1 above)
Comment (asmacdo) [reply website/content/en/docs/contribution-guidelines/developer-guide.md:7]: Good catch, this must have been missed in the initial move. Fixed.

File: website/content/en/docs/contribution-guidelines/testing/travis-build.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
-# TravisCI Build Information for Operator SDK
+---
+title: Travis
```
Comment (estroz) [inline website/content/en/docs/contribution-guidelines/testing/travis-build.md:2]:
```suggestion
title: Travis CI
```

**Pull request merged**